tmt/steps/execute/__init__.py

```diff
@@ -261,6 +262,18 @@ class ExecuteStepData(tmt.steps.WhereableStepData, tmt.steps.StepData):
         option='--duration',
         help='The maximal time allowed for the test to run.',
     )
+    ignore_duration: bool = field(
+        default=configure_bool_constant(False, 'TMT_PLUGIN_EXECUTE_TMT_IGNORE_DURATION'),
+        option='--ignore-duration',
+        is_flag=True,
+        envvar="TMT_PLUGIN_EXECUTE_TMT_IGNORE_DURATION",
+        help="""
+             Ignore test duration value and allow test to run forever.
+             Can be set by envvar even when step is not specified on the
+             commandline. This environment variable
+             will be replaced by plan.fmf or CLI arguments.
```

Comment:
```suggestion
             will be replaced by fmf config file or CLI arguments.
```

A bit better? `plan.fmf` sounds like an exact file name.